src/foam/nanos/rope/test/TestObjects.js

```diff
@@ -67,63 +67,66 @@ foam.CLASS({
   ]
 });
 
+foam.RELATIONSHIP({
+  cardinality: '*:*',
+  sourceModel: 'foam.nanos.rope.test.ROPEUser',
+  targetModel: 'foam.nanos.rope.test.ROPEUser',
+  forwardName: 'entities',
+  inverseName: 'agents',
+  junctionDAOKey: 'ropeAgentJunctionDAO'
+});
+
+foam.RELATIONSHIP({
+  cardinality: '*:*',
+  sourceModel: 'foam.nanos.rope.test.ROPEUser',
+  targetModel: 'foam.nanos.rope.test.ROPEUser',
+  forwardName: 'parteners',
+  inverseName: 'partnered',
+  junctionDAOKey: 'ropePartnerJunctionDAO'
+});
+
+foam.RELATIONSHIP({
+  cardinality: '1:*',
```

Comment:
```suggestion
  cardinality: '*:*',
```

put as *:* for now, easier to work with